src/widgets/WordListWidget.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="word-list-widget u-widget u-flex">
+    <WordList :wordList="wordList" v-if="wordList && wordList.length > 0" />
+  </div>
+</template>
+
+<script>
+  import WordList from '@/components/WordList.vue';
+  import { WIDGETS_NS } from '@/store/constants';
+
+  export default {
+    name: 'WordListWidget',
+    scaifeConfig: {
+      displayName: 'Word List',
+    },
+    components: {
+      WordList,
+    },
+    created() {
+      if (this.enabled) {
+        this.fetchData(this.url);
+      }
+    },
+    data() {
+      return {
+        wordList: [],
+      };
+    },
+    watch: {
+      passage: 'fetchData',
+    },
+    computed: {
+      enabled() {
+        // TODO: Stubbed until metadata ingestion update.
+        // return this.metadata.lang === 'grc';
+        return true;
+      },
+      metadata() {
+        return this.$store.getters[`${WIDGETS_NS}/metadata`];
+      },
+      passage() {
+        return this.$store.getters[`${WIDGETS_NS}/passage`];
+      },
+      endpoint() {
+        return 'https://vocab.perseus.org';
+      },
+      url() {
```

Comment:
I think the console errors I saw hitting https://deploy-preview-9--sv-mini.netlify.com/reader has to do with passage not being defined initially.  Perhaps we should gate `url` (to return null if passage is not defined else the constructed url`) and `fetchData` (to only fire if `url` is defined)`.

I think the `passage` watcher would then dispatch `fetchData` as expected.